client/services/AuthService.tsx

```diff
@@ -10,6 +22,7 @@ interface User {
 }
 
 const AuthService = ({ children }) => {
+  const { authenticationToken, setAuthenticationToken } = useContext(authContext)
```

Comment:
Kinda confused by this. AuthService is already a context provider. I think we just need to useState here (see `useAuth`)

Reply:
i deleted this line, not sure why it's showing here.

Reply:
shows as outdate, nvm